### DISCUSSION TITLE
How are checksums saved?

### DISCUSSION THREAD

**Discussion by EmmanuelMess** [probonopd/irdb#99] [Q&A]
How does this database save the checksum algorithms for Air Conditioner messages?

**Comment by probonopd** [comment]
I am not sure I understand what you mean by "checksum algorithms".

The protocol/device/subdevice/function notation is a way to specify the exact IR code that should be sent to control a particular electronic device. By using this notation, software like [HARCToolbox](http://www.harctoolbox.org/) by @bengtmartensson can generate the correct IR code for a given device and function (presumably including any checksums, should they be needed), allowing you to control it using a compatible IR transmitter.
